cmd/migration.go

```diff
@@ -149,6 +151,8 @@ func init() {
 	migrationFetchCmd.MarkFlagsMutuallyExclusive("db-url", "linked", "local")
 	migrationCmd.AddCommand(migrationFetchCmd)
 	// Build new command
+	newFlags := migrationNewCmd.Flags()
+	newFlags.BoolVarP(&repeatable, "repeatable", "r", false, "Creates a repeatable migration instead of a common migration.")
```

Comment:
```suggestion
	newFlags.BoolVarP(&repeatable, "repeatable", "r", false, "Creates a repeatable migration instead of a versioned migration.")
```

Use consistent terminology as flyway